### PR TITLE
[dom-gpu] Amber 17.08

### PR DESCRIPTION
**Pull request by victorusu**

### CODE REVIEW

File: easybuild/easyconfigs/a/Amber/Amber-16-2017.08-CrayGNU-17.08-cuda-8.0.eb [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+versionsuffix = '-cuda-%s' % cudaversion
+amberversion = '16'
+
+homepage = 'http://ambermd.org/'
+description = 'Assisted Model Building with Energy Refinement'
+
+toolchain = {'name': 'CrayGNU', 'version': '17.08'}
+toolchainopts = { 'verbose' : False }
+#sources = ['Amber%(version)s.tar.bz2', 'AmberTools%(version)s.tar.bz2']
+sources = ['Amber%(version)s.tar.bz2']
```
Comment (lucamar) [inline easybuild/easyconfigs/a/Amber/Amber-16-2017.08-CrayGNU-17.08-cuda-8.0.eb]: Could you please use the absolute path here?

Comment (gppezzi) [inline easybuild/easyconfigs/a/Amber/Amber-16-2017.08-CrayGNU-17.08-cuda-8.0.eb]: Shouldn't you use the absolute path here?

File: easybuild/easyconfigs/a/Amber/Amber-16-2017.08-CrayGNU-17.08-parallel.eb [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+versionsuffix = '-parallel'
+amberversion = '16'
+
+homepage = 'http://ambermd.org/'
+description = 'Assisted Model Building with Energy Refinement'
+
+toolchain = {'name': 'CrayGNU', 'version': '17.08'}
+toolchainopts = { 'verbose' : False }
+#sources = ['Amber%(version)s.tar.bz2', 'AmberTools%(version)s.tar.bz2']
+sources = ['Amber%(version)s.tar.bz2']
```
Comment (gppezzi) [inline easybuild/easyconfigs/a/Amber/Amber-16-2017.08-CrayGNU-17.08-parallel.eb]: Shouldn't you use the absolute path here?

**Comment by victorusu** [comment]
@lucamar, The 2017.08 is not an additional versioning, but the de facto versioning of Amber for us. It means that it has all patches up to August 2017. Since Amber does not have any minor version releases.

**Comment by victorusu** [comment]
It did not pass because of some CI issue, not due to the easybuild recipes.

**Comment by lucamar** [comment]
retest this please

**Comment by lucamar** [comment]
retest this please

**Comment by lucamar** [comment]
`ERROR: Irresolvable dependencies encountered: bzip2/.1.0.6-CrayGNU-17.08, zlib/.1.2.11-CrayGNU-17.08, flex/.2.6.0-CrayGNU-17.08`

**Comment by victorusu** [comment]
Yes, the CI should install Amber with the `-r` option.

**Comment by lucamar** [comment]
@victorusu Jenkins project `TestingEB`calls `production.sh`, which uses `-r`by default: please cross check the output. 
The problem is that the EasyBuild does not find the configuration files of the dependencies, since they are not included in this branch.

**Comment by victorusu** [comment]
retest this please

**Comment by victorusu** [comment]
retest this please

**Comment by victorusu** [comment]
retest this please

**Comment by victorusu** [comment]
retest this please

File: easybuild/easyconfigs/a/Amber/Amber-16-2017.08-CrayGNU-17.08-cuda-8.0.eb [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+    ('flex', '2.6.4'),
+]
+
+buildininstalldir = True
+
+prebuildopts = 'cd .. && mv amber%s/* . && ' % amberversion
+prebuildopts += 'export AMBERHOME=%(builddir)s; export CUDA_HOME=$CUDATOOLKIT_HOME;'
+# Patching from David Case should be applied after updating
+# updating
+#prebuildopts += './update_amber --update && '
```
Comment (jgphpc) [inline easybuild/easyconfigs/a/Amber/Amber-16-2017.08-CrayGNU-17.08-cuda-8.0.eb]: not needed ?

Comment (victorusu) [reply easybuild/easyconfigs/a/Amber/Amber-16-2017.08-CrayGNU-17.08-cuda-8.0.eb]: not anymore

**Comment by victorusu** [comment]
retest this please

**Comment by victorusu** [comment]
retest this please

**Comment by victorusu** [comment]
retest this please

**Pull request merged**